test/public-types/checkbox-field.ts

```diff
@@ -0,0 +1,6 @@
+// ./checkbox-field
+import { Checkbox } from '@base-ui-components/react/checkbox';
```

Comment:
I guess we have the same problem if we do:


```suggestion
import { type Checkbox } from '@base-ui-components/react/checkbox';
```